lib/cli/utils.ts

```diff
@@ -80,3 +80,12 @@ export const waitForCert = (certPath: string) => {
     }
   });
 };
+
+export const argChecks = {
```

Comment:
I don't think we need the whole `argChecks` thing here, which I intended as a set of reusable server side argument checks for similar parameters across multiple calls.

I think instead we should check if `argv.amount === 'all'` as we already do, else check `!isNan()`. If it's not a number and not `all`, then `console.error` a message and `process.exit(1)` which is how we handle similar situations for other commands.

Reply:
I put it separately just as a beginning for all command checks wouldn't it be better to move all arg checks here? As we already do in `Service.ts` ?

Reply:
Do you mean move all argument checks from the server to the cli? That's a major refactor and also would mean that other consumers of the xud API would have to do their own arg checking. I think it makes sense to leave the majority of these checks on the server side, and have the cli tool just ensure that it is using the correct types for all the parameters (we'd get ugly grpc errors otherwise). This is sort of a special case where we're trying to have command line arg assign values to two separate request parameters (`all` and `amount`).

Reply:
Alright gotcha!